src/main/java/org/openstreetmap/atlas/geography/atlas/packed/PackedAtlas.java

```diff
@@ -78,85 +85,89 @@
     protected static final String FIELD_SERIALIZER = "serializer";
     protected static final String FIELD_SAVE_SERIALIZATION_FORMAT = "saveSerializationFormat";
     protected static final String FIELD_LOAD_SERIALIZATION_FORMAT = "loadSerializationFormat";
+    protected static final String FIELD_CONTAINS_ENHANCED_RELATION_GEOMETRY = "containsEnhancedRelationGeometry";
     protected static final String FIELD_META_DATA = "metaData";
     protected static final String FIELD_DICTIONARY = "dictionary";
-    protected static final Object FIELD_DICTIONARY_LOCK = new Object();
+    private final transient Object fieldDictionaryLock = new Object();
```

Comment:
I am curious about the reasoning behind those changes!

Reply:
when static objects are used as locks, any threads that try to deserialize use the same monitors even if they're deserializing different Atlases. that is ... not the intended behavior of these monitors, and has serious performance impact.